parser-library/nt-headers.h

```diff
@@ -55,6 +55,49 @@ const std::uint16_t DIR_IAT = 12;
 const std::uint16_t DIR_DELAY_IMPORT = 13;
 const std::uint16_t DIR_COM_DESCRIPTOR = 14;
 
+// Machine Types
```

Comment:
I think these constants should be made `static` or `constexpr` (or both), as they are global but defined in a header.